x-pack/metricbeat/module/aws/cloudwatch/_meta/docs.asciidoc

```diff
@@ -9,4 +9,5 @@ Some specific AWS permissions are required for IAM user to collect AWS SQS metri
 ec2:DescribeRegions
```

Comment:
Not from this PR, but previous paragraph mentions SQS, I guess this should mention cloudwatch instead?

Reply:
Ha yes, done! Good catch!